cosmwasm/cli-utils.js

```diff
@@ -166,8 +166,8 @@ const addMigrateOptions = (program) => {
 };
 
 const addProposalOptions = (program) => {
-    program.addOption(new Option('-t, --title <title>', 'title of proposal'));
-    program.addOption(new Option('-d, --description <description>', 'description of proposal'));
+    program.addOption(new Option('-t, --title <title>', 'title of proposal').makeOptionMandatory(true));
+    program.addOption(new Option('-d, --description <description>', 'description of proposal').makeOptionMandatory(true));
```

Comment:
Generate the title and description automatically instead? This was an improvement we wanted regardless